.github/workflows/snapshot-nhsn-data.yml

```diff
@@ -2,7 +2,7 @@ name: Snapshot NHSN data and upload to S3
 
 on:
   schedule:
-    - cron: "45 5 * * 3" # every Wednesday at 5:45PM UTC == 12:45PM EST
+    - cron: "45 17 * * 3" # every Wednesday at 5:45PM UTC == 12:45PM EST
```

Comment:
5pm --> 17.  Last week this ran in the early hours.

Reply:
Looks good, but it's hard to know whether the cron is specifying UTC or local...